8.x/core-concepts/cruds.md

```diff
@@ -2,99 +2,55 @@
 
 [[toc]]
 
-This document reviews how to implement CRUD (create, read, update, delete) in Larammerce.
-As you know, it is not only related to Larammerce and is related to the structure of the Laravel framework.
-In larammerce, tools have been created to make implement in CRUD easier. Also, the created CRUD should be the same as the previous CRUDs of the system, so it should be followed the guidelines provided in this document.
+This document reviews how to implement `CRUD` (create, read, update, delete) in the Larammerce platform. A `ToDoList` project is done step by step to illustrate the operational concepts.
 
-In this tutorial, the principles of CRUD operation in Larammerce will be taught, so, let's follow a few steps to create it:
+## Make migrations
 
-**Step 1**: Create Migration
+Larammerce is a schema-first platform. So you must create a table for your todos.
 
-**Step 2**: Add Model
+Run the following command in the CMD:
```

Comment:
Let’s say terminal or command line instead of CMD because cmd is just a command shell related to windows.